todo_app/data/trello_items.py

```diff
@@ -0,0 +1,70 @@
+# create a class
+
+import requests
```

Comment:
You're using this library but it hasn't been added to your project dependencies, which means future users (like me!) have to add it manually. Instead, we should add it with `poetry add requests` and then future users will pick it up automatically when they run `poetry install`

Reply:
done